agent/handlers/v1/response.go

```diff
@@ -67,6 +67,7 @@ type PortResponse struct {
 	ContainerPort uint16 `json:"ContainerPort,omitempty"`
 	Protocol      string `json:"Protocol,omitempty"`
 	HostPort      uint16 `json:"HostPort,omitempty"`
+	HostIp        string `json:"HostIp,omitempty"`
```

Comment:
nit: ```HostIP```?

Reply:
Docker inspect response has `HostIp` so was trying to be consistent with that, but I have no preference.
```
Ports": {
                "80/tcp": null,
                "8888/tcp": [
                    {
                        "HostIp": "0.0.0.0",
                        "HostPort": "49825"
                    },
                   ...
```

Reply:
oh weird. lets keep it consistent with the docker inspect response then, thanks!